js_modules/dagit/packages/core/src/ui/MenuLink.tsx

```diff
@@ -16,6 +22,9 @@ interface MenuLinkProps
 export const MenuLink: React.FC<MenuLinkProps> = (props) => {
   const {icon, intent, text, ...rest} = props;
 
+  if (rest.disabled) {
+    return <MenuItem disabled icon={icon} intent={intent} text={text} />;
```

Comment:
Alternatively, should this just use `...rest`?

Reply:
Ahh I tried, but it gets angry about onClick and a few others having the wrong type (which is ok really, since it's disabled). Let me see what I can do!